sdk/storage/storage-blob/src/AppendBlobClient.ts

```diff
@@ -123,14 +154,31 @@ export interface AppendBlobAppendBlockFromURLOptions {
    */
   sourceModifiedAccessConditions?: Models.ModifiedAccessConditions;
   /**
-   * A Uint8Array holding the MD5 hash of the source block content.
-   * It is only used to verify the integrity of the block during transport.
-   * It is not stored in with the blob.
+   * An MD5 hash of the append block content from the URI.
+   * This hash is used to verify the integrity of the append block during transport of the data from the URI.
+   * When this is specified, the storage service compares the hash of the content that has arrived from the copy-source with this value.
+   *
+   * sourceContentMD5 and sourceContentCrc64 cannot be set at same time.
    *
    * @type {Uint8Array}
    * @memberof AppendBlobAppendBlockFromURLOptions
    */
   sourceContentMD5?: Uint8Array;
+  /**
+   * A CRC64 hash of the append block content from the URI.
+   * This hash is used to verify the integrity of the append block during transport of the data from the URI.
+   * When this is specified, the storage service compares the hash of the content that has arrived from the copy-source with this value.
+   *
+   * sourceContentMD5 and sourceContentCrc64 cannot be set at same time.
+   */
```

Comment:
Minor: add the @type and @memberof info, I find these are added for track 2, would better to make this consistent if this is convention for track 2.

Reply:
Added here. 

I'll create a new issue for JSDoc consistencies across the storage packages which will be tackled post preview 3.

Reply:
https://github.com/Azure/azure-sdk-for-js/issues/5098